app/src/main/res/layout/activity_create_account.xml

```diff
@@ -4,16 +4,24 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="com.wire.android.MainActivity">
+    android:background="@color/authentication_background">
 
-    <TextView
-        android:id="@+id/hello"
+    <ImageButton
+        android:id="@+id/createAccountBackButton"
+        style="@style/ImageButton.Back"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello Wire Android Reloaded!"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:elevation="1dp"
+        android:padding="@dimen/back_button_padding"
         app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="ContentDescription" />
```

Comment:
Instead of ignoring the warning, do we want to start considering adding content descriptions to aid accessibility in this new project?

Reply:
I put the contentDescription in the `ImageButton.Back` style, but A.S. cannot read inside the style and complains that there isn't one. As an alternative, I can move the contentDescription here

Reply:
It's clever, but, It might be better placed here to be honest, I personally don't think it's the responsibility of the styles. We can put it to the team for a discussion, but, ignoring this warning seems like an avoidable overhead here.

Reply:
fair enough. done 👍